week-4/C-more-complex-objects/exercise-2.js

```diff
@@ -28,15 +28,22 @@ var newCurrentOwner = {
 // - assign the value of the variable 'newCurrentOwner' as the value to the house's "currentOwner"
 // - from the list of previous owners, replace only "John A." with "Stephen B."
 // - give the house a new property called 'isForSale' with the value 'false'
-
-
-
-
+newCurrentOwner.firstName = "Margaret";
+newCurrentOwner.lastName = "Conway";
+newCurrentOwner = house.currentOwner;
+house.previousOwners = ["Claire M.", "Stephen B."];

```

Comment:
`replace ONLY "John A." with "Stephen B."`  Can you think of a way to do this?

Reply:
Aham, I've changed the name by assigning the array index.